client-spark/spark3/src/main/java/org/apache/spark/shuffle/writer/RssShuffleWriter.java

```diff
@@ -109,7 +113,10 @@ public RssShuffleWriter(
       RssShuffleManager shuffleManager,
       SparkConf sparkConf,
       ShuffleWriteClient shuffleWriteClient,
-      RssShuffleHandle<K, V, C> rssHandle) {
+      RssShuffleHandle<K, V, C> rssHandle,
+      ShuffleHandleInfo shuffleHandleInfo,
+      TaskContext context,
+      ShuffleManagerClient shuffleManagerClient) {
```

Comment:
@yl09099  Why change this constructor to add TaskContext and ShuffleManagerClient?

Reply:
Is it possible to put shuffleHandleInfo into rssHandle? This will make the code cleaner. @yl09099

Reply:
> @yl09099 Why change this constructor to add TaskContext and ShuffleManagerClient?



> TaskContext

ShuffleManagerClient has been removed, from which TaskContext's later PR needs to obtain information such as Stage AttemptNumber.

Reply:
> Is it possible to put shuffleHandleInfo into rssHandle? This will make the code cleaner. @yl09099

To put shuffleHandleInfo object into the broadcast on the Driver side, if you want to put shuffleHandleInfo into rssHandle, you need to build RPC interface through the Driver's broadcast object to put in, here I think it is a waste of performance.